### PR TITLE
fail2ban multiple fixes

### PR DESCRIPTION
**Pull request by gsanchietti**
Work in progress: cleanup failban 10 PR.

See also NethServer/nethserver-firewall-base#121

**WARNING**: original fail2ban0.10 should be rebased before merge.

### CODE REVIEW

File: root/etc/fail2ban/filter.d/urbackup-auth.conf [hunk 1]
```diff
@@ -2,5 +2,5 @@
 #this filter is made against brute force attack to the server of urbackup
 # Author Stephane de Labrusse <stephdl@de-labrusse.fr>
 
-failregex = ^<HOST> - - \[*\] "POST /{${'urbackup-server'}{'Name'}||'urbackup'}/x\?a\=login HTTP/1.1" 200 33$
+failregex = ^<HOST> - - \[*\] "POST /urbackup/x\?a\=login HTTP/1.1" 200 33$
```
Comment (stephdl) [inline root/etc/fail2ban/filter.d/urbackup-auth.conf:5]: sysadmin can change the /urbackup, if you do that the jail won't catch it

Comment (gsanchietti) [reply root/etc/fail2ban/filter.d/urbackup-auth.conf:5]: The file is now marked as configuration, the user can manually change it.

Comment (stephdl) [reply root/etc/fail2ban/filter.d/urbackup-auth.conf:5]: yes but it is manual, sysadmin won't go to the terminal

Comment (gsanchietti) [reply root/etc/fail2ban/filter.d/urbackup-auth.conf:5]: The fail2ban must not have access to urbbackup internals (like the prop `name`).
If this is still needed, the relevant code should be moved to a template inside the urbbackup package itself.

**Comment by gsanchietti** [comment]
Replaced by #2

**Pull request closed without merging**